x/qgb/types/errors.go

```diff
@@ -5,20 +5,20 @@ import (
 )
 
 var (
-	ErrDuplicate                                 = errors.Register(ModuleName, 2, "duplicate")
-	ErrEmpty                                     = errors.Register(ModuleName, 6, "empty")
-	ErrNoValidators                              = errors.Register(ModuleName, 12, "no bonded validators in active set")
-	ErrInvalidValAddress                         = errors.Register(ModuleName, 13, "invalid validator address in current valset %v")
-	ErrInvalidEVMAddress                         = errors.Register(ModuleName, 14, "discovered invalid EVM address stored for validator %v")
-	ErrInvalidValset                             = errors.Register(ModuleName, 15, "generated invalid valset")
-	ErrAttestationNotValsetRequest               = errors.Register(ModuleName, 16, "attestation is not a valset request")
-	ErrAttestationNotFound                       = errors.Register(ModuleName, 18, "attestation not found")
-	ErrNilAttestation                            = errors.Register(ModuleName, 22, "nil attestation")
-	ErrUnmarshalllAttestation                    = errors.Register(ModuleName, 26, "couldn't unmarshall attestation from store")
-	ErrNonceHigherThanLatestAttestationNonce     = errors.Register(ModuleName, 27, "the provided nonce is higher than the latest attestation nonce")
-	ErrNoValsetBeforeNonceOne                    = errors.Register(ModuleName, 28, "there is no valset before attestation nonce 1")
-	ErrDataCommitmentNotGenerated                = errors.Register(ModuleName, 29, "no data commitment has been generated for the provided height")
-	ErrDataCommitmentNotFound                    = errors.Register(ModuleName, 30, "data commitment not found")
-	ErrLatestAttestationNonceStillNotInitialized = errors.Register(ModuleName, 31, "the latest attestation nonce has still not been defined in store")
-	ErrInvalidDataCommitmentWindow               = errors.Register(ModuleName, 32, "invalid data commitment window")
+	ErrDuplicate                                 = errors.Register(ModuleName, 1002, "duplicate")
+	ErrEmpty                                     = errors.Register(ModuleName, 1006, "empty")
+	ErrNoValidators                              = errors.Register(ModuleName, 1012, "no bonded validators in active set")
+	ErrInvalidValAddress                         = errors.Register(ModuleName, 1013, "invalid validator address in current valset %v")
```

Comment:
I think that we can go for sequential error codes because the reason we're jumping from 1002 to 1006 for example, is because we deleted the corresponding error...

Also, @evan-forbes I thought that the error codes are defined per module. I don't see how these overlap with other errors. Or I am missing something

Reply:
> I think that we can go for sequential error codes because the reason we're jumping from 1002 to 1006 for example, is because we deleted the corresponding error...

Got it, as I never thought of it from this angle, ie, there can be jumping between errors(1002 to 1006),
I thought of keeping it simple so that there can be a flow between the previous and updated one.

Reply:
I think that it's okey to do it now since we don't need to worry about backward compatibility as we still didn't launch mainnet. But after mainnet, definitly, the way you did it is the right way.
@evan-forbes What do you think? should we keep the jumps?

Reply:
agree, we should probably go in order here.